iamproperty/iamproperty.github.io
# Create icons page and start building up the icon library

## Summary of Changes:
1. Build an edit file to export SVGs out of illustrator
2. Build an SVG sprite for the icons
3. Create the documentation around the icons

## Rationale
Make using icons really simple and consistent. Create the ability to modify the icons colours and line thickness for when we need to apply things like hover states.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /foundations/icons

## Ticket(s)
DS-31

## Checklist

## Notes
There are 75 icons that will need to be created in this way.
